test/test_suite.py

```diff
@@ -837,3 +838,72 @@ def test_nolength_binary(self):
 @pytest.mark.skip("Spanner doesn't support quotes in table names.")
 class QuotedNameArgumentTest(_QuotedNameArgumentTest):
     pass
+
+
+class TextTest(_TextTest):
+    @pytest.mark.skip("Spanner throws an error")
+    def test_text_empty_strings(self, connection):
+        """
+        SPANNER OVERRIDE:
+
+        Spanner DBAPI throws an error when cleanup tried to
+        rollback the connection after the test executed successfully.
+        The error is `ValueError: Transaction is already rolled back`.
```

Comment:
would it be better to either skip the roll back by mocking it or overriding that part. The original test case seem valid and fine: https://github.com/sqlalchemy/sqlalchemy/blob/ea225c63f98dc3c250cee1b243762970b8e0932a/lib/sqlalchemy/testing/suite/test_types.py#L219